user_guide_src/source/libraries/validation.rst

```diff
@@ -201,6 +201,42 @@ Then add validation rules in the controller (**Form.php**)::
 
 If you submit the form you should see the success page or the form with error messages.
 
+Config for Validation
+*********************
+
+.. _validation-traditional-and-strict-rules:
+
+Traditional and Strict Rules
+============================
+
+CI4 has two kind of Validation rule classes.
+The default rule classes (**Traditional Rules**) have the namespace ``CodeIgniter\Validation``,
+and the new classes (**Strict Rules**) have ``CodeIgniter\Validation\StrictRules``, which provide strict validation.
+
+The **Traditional Rules** implicitly assume that string values are validated.
+And the input value may be converted implicitly to a string value.
```

Comment:
Do not start your sentence with And. You may add this to the previous sentence, or remove "and".

Reply:
Fixed.